databricks/koalas/tests/test_series.py

```diff
@@ -1599,3 +1599,11 @@ def test_unstack(self):
     def test_item(self):
         kser = ks.Series([10, 20])
         self.assertRaises(ValueError, lambda: kser.item())
+
+    def test_filter(self):
+        kser = ks.Series([0, 1, 2], index=["one", "two", "three"])
+        pser = kser.to_pandas()
+
+        self.assert_eq(pser.filter(items=["one", "three"]), kser.filter(items=["one", "three"]))
+        self.assert_eq(pser.filter(regex="e$"), kser.filter(regex="e$"))
+        self.assert_eq(pser.filter(like="hre"), kser.filter(like="hre"))
```

Comment:
Could you add a test with `MultiIndex` ?

Reply:
Of course, I will also add test with `Mutiindex`

Reply:
Thanks, and I think we'd better to have error test when axis=1 for `Series`.

like 

```python
        with self.assertRaisesRegex(ValueError, "Series does not support columns axis."):
            kser.filter(like="hre", axis=1)
```

Reply:
@itholic 
The `filter` function of Koalas `DataFrame` did not support `Multiindex`. So, I am currently posting PR by adding `Multiindex` support. (https://github.com/databricks/koalas/pull/1512) Test cases related to `Multiindex` will be added together in this PR.

Reply:
Got it !